doxygen/src/components_schema.txt

```diff
@@ -593,20 +593,24 @@ A component describes a collection of files (source, header, configuration, libr
     <td>xs:string </td>
     <td>0..1 </td>
   </tr>
-
   <tr>
     <td>\ref element_files "files"</td>
     <td>Grouping element for all files that are part of this component.</td>
     <td>group</td>
     <td>1</td>
   </tr>
-
   <tr>
     <td>\ref element_extensions "extensions"</td>
     <td>Grouping element for all extensions concerning this component.</td>
     <td>group</td>
     <td>1</td>
   </tr>
+  <tr>
```

Comment:
We had previously agreed to add the "<extensions>" sub-element to the `<component>` element: https://open-cmsis-pack.github.io/Open-CMSIS-Pack-Spec/main/html/pdsc_components_pg.html#element_extensions.
```
Release Notes: 
1. June 2022: v1.7.8
   - added extension points description for components as suggested by STMicroelectronics
```
Do we need both? What has changed in between?

Reply:
It would be good to see real examples for both elements I guess.